hack/make-rules/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])
+
+# TODO: These "sh_binary" targets" should actually be "sh_tests".
+# sh_binary rules introduced on top of legacy 'make' to enforce
+# dependency sanity checks.  More work required to refactor to use
+# bazel sandbox.
+
+sh_binary(
+    name = "test-federation-cmd",
+    srcs = ["test-federation-cmd.sh"],
+    deps = [
+        ":make-rules",
+        "//cluster/lib",
+        "//hack/lib",
+    ],
+)
+
+sh_binary(
```

Comment:
I'd like to, but it's too much work for this PR.
Modified the TODO to point to an issue against which the work can be done.